### PR TITLE
Cypress: Add Volume cases

### PR DESCRIPTION
**Pull request by DaiYuzeng**
#### Releated cases:
- https://harvester.github.io/tests/manual/volumes/create-image-from-volume/
- https://harvester.github.io/tests/manual/volumes/create-volume-root-disk-vm-image-form/
- https://harvester.github.io/tests/manual/volumes/create-volume-root-disk-vm-image-form-label/
- https://harvester.github.io/tests/manual/volumes/delete-volume-that-is-not-attached-to-vm/
- https://harvester.github.io/tests/manual/volumes/delete-volume-that-was-attached-to-vm-but-is-not-now/

### CODE REVIEW

File: cypress/pageobjects/virtualmachine.po.ts [hunk 1]
```diff
@@ -180,7 +184,7 @@ export class VmsPage {
     this.save();
   }
 
-  public delete(name: string) {
+  public delete(name: string, { removeRootDisk }: { removeRootDisk?: boolean } = { removeRootDisk: true }) {
```
Comment (DaiYuzeng) [inline cypress/pageobjects/virtualmachine.po.ts:187]: @n313893254 I add the `removeRootDisk` parameter on VM delete function. Please take a look, does it make sense?

Comment (n313893254) [reply cypress/pageobjects/virtualmachine.po.ts:187]: LGTM

**Comment by noahgildersleeve** [comment]
There is a very large PR about to be merged into main that is going to affect this PR @DaiYuzeng . It's in https://github.com/harvester/tests/pull/255 . I was reviewing this PR and I noticed that a lot of my comments were going to be things that were already in that PR. You might want to rebase off of that PR first then resubmit this PR. If you want me to I can still proceed with the review, but I think it will save you the time of having to then fix conflicts after that PR is merged. Let me know what you want to do.

**Comment by DaiYuzeng** [comment]
@noahgildersleeve I would like to rebase #255 first to resolve the conflicts and then resumit this PR, thanks!

**Pull request closed without merging**